### PR TITLE
Added usage hint for --auto-sampled-textures and added test cases

### PR DESCRIPTION
**Pull request by Rob2309**
This PR adds the requested test cases and usage hints for #2616.

### CODE REVIEW

File: StandAlone/StandAlone.cpp [hunk 1]
```diff
@@ -1819,6 +1819,8 @@ void usage()
            "                                    without explicit bindings\n"
            "  --auto-map-locations | --aml      automatically locate input/output lacking\n"
            "                                    'location' (fragile, not cross stage)\n"
+           "  --auto-sampled-textures           Removes sampler variables and converts\n" 
+           "                                    existing textures to combined image-samplers\n"
```
Comment (greg-lunarg) [inline StandAlone/StandAlone.cpp:1823]: ...combined texture-samplers?

**Review by greg-lunarg — CHANGES_REQUESTED** [review]
One question about terminology. Otherwise good. Thanks for the submission!

File: StandAlone/StandAlone.cpp (hunk 1 above)
Comment (Rob2309) [reply StandAlone/StandAlone.cpp:1823]: Oops I used the Spirv terminology there, I could change this when I‘m home.

Comment (greg-lunarg) [reply StandAlone/StandAlone.cpp:1823]: or even ...sampled textures :)

Comment (Rob2309) [reply StandAlone/StandAlone.cpp]: done :)

**Pull request merged**